pkg/scheduler/objects/application.go

```diff
@@ -137,6 +147,37 @@ func NewApplication(siApp *si.AddApplicationRequest, ugi security.UserGroup, eve
 	return app
 }
 
+// Adjust the placeholder timeout if "creationTime" is defined. It's used during Yunikorn restart,
+// in order to properly track how long a placeholder pod should be in "Running" state.
+func (sa *Application) adjustPlaceholderTimeout(placeholderTimeout time.Duration, siApp *si.AddApplicationRequest) (bool, time.Duration) {
+	creationTimeTag := siApp.Tags["creationTime"]
```

Comment:
As I mentioned in https://github.com/apache/yunikorn-k8shim/pull/405/, this is a tag, but `AddApplicationRequest` can be extended with a `creationTime` field.